cleverhans/utils_tf.py

```diff
@@ -516,3 +516,17 @@ def get_available_gpus():
   """
   local_device_protos = device_lib.list_local_devices()
   return [x.name for x in local_device_protos if x.device_type == 'GPU']
+
+
```

Comment:
maybe put this somewhere like a new `cleverhans.initializers`. I think `utils_tf` should be more about just making tensorflow run, rather than implementing specific ML feature in tensorflow.

Reply:
+ 1 

Can we also move the initializer from `cleverhans_tutorials/tutorial_models.py`

Reply:
Nevermind Ian wrote that same comment below